_data/accessibility-tests/table.yml

```diff
@@ -0,0 +1,108 @@
+title: Table
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey meaning in a table.
+    summary_additional: |
+      When viewing the table,
+      you never have to rely on color alone to understand the table's meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.6.0
+    wcag_criterion: 1.4.1
+  - summary: Table content meets color contrast requirements.
+    summary_additional: |
+      When you view the content in a table and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the table background and text is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.6.0
+    wcag_criterion: 1.4.3
+# Zoom/screen magnification tests
+  - summary: Table content is visible and functional when screen is magnified.
+    summary_additional: |
+      When you zoom the screen to 200%,
+      content in the table remains visible and does not overlap with other content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.6.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Table elements can be navigated by keyboard without special cues.
+    summary_additional: |
+      When using a keyboard to navigate a table,
+      a user can navigate using arrow keys or the tab key with no unexpected cues or instruction.
+    test_status: exception
+    status_details: |
+      The scrollable table does not scroll with keystrokes in JAWS.
+      A user would need to use their mouse to follow along with the focus indicator in JAWS.
+    test_type: keyboard
```

Comment:
> [!Note]
> Note to self: need to add a link to the related issue.

Reply:
Added link to related issue in [a675795](https://github.com/uswds/uswds-site/pull/2538/commits/a6757952b668c94cf170bd2bf0213432dc134a50)